Dockerfile

```diff
@@ -33,6 +33,9 @@ COPY --from=builder /go/src/github.com/tikv/pd/bin/pd-ctl /pd-ctl
 COPY --from=builder /go/src/github.com/tikv/pd/bin/pd-recover /pd-recover
 COPY --from=builder /jq /usr/local/bin/jq
 
+RUN apk add --no-cache \
```

Comment:
Seem we have already added it in line 7?

Reply:
I think line 7 means builder image install curl, but real pd image not install curl command.

Reply:
This is similar to the dockerfile of tidb, you can refer to it: https://github.com/pingcap/tidb/blob/6.2.0/Dockerfile